openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -552,8 +552,8 @@ def _process_texture(self, filepath, resource,
                         self.log.warning("cannot guess the colorspace"
                                          "color conversion won't be available!")    # noqa
 
-
-            additional_args.extend(["--colorconfig", config_path])
+            if config_path:
```

Comment:
The variable `config_path` will still be undefined. You'd need to initialize it above to `config_path = None` to make this work

Reply:
Yes, I just find out about that.